inst/rmd/qc_report.rmd

```diff
@@ -72,10 +72,30 @@ has_cite <- "CITEseq" %in% altExpNames(filtered_sce)
 if (has_cite){
   modalities <- c(modalities, "CITE-seq")
 }
+
+has_multiplex <- "cellhash" %in% altExpNames(filtered_sce)
+if(has_multiplex){
+  modalities <- c(modalities, "Multiplex")
+  multiplex_samples <- paste(metadata(filtered_sce)$sample_id, collapse = ", ")
```

Comment:
What do you mean it only shows a single sample? We process a single QC report for each library and so we actually are showing the single library ID here and passing that in as the `sample_id` and not the `scpca_sample_id`. So I think it makes sense to show just the single library ID here. Perhaps we should update this so that we continue to pass in the library ID , but have both the library ID and the sample ID (since we can now grab the sample ID from the metadata of the SCE) displayed in the table?

Reply:
Oh you are right! I was confused by the "sample" label. We should probably change the name of that param & variable to reflect that they are library ids. Having the actual sample id in the table as well seems like it would be a good idea!

Reply:
Yes I think that all sounds good! Will make those changes throughout to reflect that it's a library ID. We will also need to remember to update the script that generates the QC report in `scpca-nf` when we change the parameter name.